### PR TITLE
Added post for creating a Jekyll Title Banner

### PR DESCRIPTION
**Pull request by maxx1128**
Added another Jekyll tutorial post, this one about making a popular pattern I call the "Title Banner." Includes a few images as well as code snippets.

### CODE REVIEW

File: _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title:  "Creating an Title Banner in Jekyll"
```
Comment (danbernier) [inline _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown]: Typo: "an" -> "a"

File: _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+categories: jekyll featured banner front-end tutorial
+author: Max Antonucci
+
+source: internal
+ext-url: none
+---
+
+Today there's lots of popular front-end patterns: the off-canvas menu, cards, flexbox media objects, just to name a few. One of the most popular I've seen, and am still seeing more of, is what I call the "Title Banner" (I apologize if there's a more official name out there I didn't find). It's popular for very good reasons and many devs will find it useful. This is a quick tutorial on making one for a Jekyll site.
+
+### Back up. What is an Title Banner?
```
Comment (danbernier) [inline _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown]: Same typo: "an" -> "a"

File: _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+The quickest, most effective way I know is otherwise frowned upon in front-end: inline styles in the HTML - one of the few exceptions I have for this. So in the `post.html` layout, one needs to add this:
+
+{% highlight html %}
+<div class="FeaturedImgBanner" {%raw%}{% if page.featured-img %} style="background-image: url('{{ page.featured-img }}');" {% endif %}{%endraw%}>
+    <!-- Include your post title, byline, date, and other info inside the header here. -->
+</div>
+{% endhighlight %}
+
+Viola, we now have a post template that will include any image one choses as a background image. Some extra CSS is needed for display reasons, but since it's the same for each banner, it can be used in the regular stylesheet.
```
Comment (danbernier) [inline _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown]: Typo: "Viola" should be "Voilà", unless you're going for the pun :)

File: _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  background-position: center;
+  background-image: url(/path/to/backup/image); // Your backup background image
+}
+{% endhighlight %}
+
+### What if the image and text don't contrast enough?
+
+I think this is one of the biggest reasons people decide against a Title Banner: if the image and the text are similarly colored, the text can't be read. It's hard to read white text against an image of a snowstorm after all.
+
+The hard solution is take and edit photos specifically with contrast in mind, which can really limit the options of photography-illiterate devs0. The second is to guarantee there'll be contrast with a little CSS.
```
Comment (danbernier) [inline _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown]: I _think_ "devs0" is a typo?

File: _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+</div>
+{% endhighlight %}
+
+Now any post can have a light _or_ dark image contrast!
+
+#### In Conclusion
+
+The Title Banner is a popular pattern, and for good reason. It's stylish, uses less space, and improves user experience. Title Banners that can work with lots of different images can find lots of use on any website, personal and professional.
+
+Even if you don't code with Jekyll, the basic way it's executed here can be carried over for almost any other CMS. So for front-end devs, it's a pattern with getting familiar with.
```
Comment (danbernier) [inline _posts/2016-10-26-creating-featured-image-banner-jekyll.markdown]: Typo: "it's a pattern ~~with~~ _worth_ getting familiar with."

**Comment by danbernier** [comment]
Looks perfect to me, @maxx1128. Merge away, I say.

**Comment by maxx1128** [comment]
@sukima Thank you! I was worried writing about one pattern wouldn't give enough to write about, but I'm glad you enjoyed it.

@danbernier Thanks for pointing these out, just went through and corrected them really fast.

**Comment by maxx1128** [comment]
Thank you both! Just made the merge, and scheduled to tweet the link to it tomorrow at 9:05am.

**Pull request merged**